code_examples/core_features/signCallback/signExtrinsicCallback.ts

```diff
@@ -0,0 +1,19 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { lookupDidDocument, lookupDidKeyPair } from '.'
```

Comment:
This should not show up in the user-visible code, so it should perhaps be wrapped in a TsJsSnippet, and not a TsJsBlock, so that we can hide these imports. Alternatively, the functions should be declared in the file, as otherwise the user has not way to know what these functions do.

Reply:
I think it's fine to show up. It needs to be clear where these methods come from. I think it's not clear enough that the user has to implement them themself.
I tried to make it clearer by explaining everything step by step. But i would also maybe add a disclaimer that these methods need to be implemented by the dev or an extension.

[0036481](https://github.com/KILTprotocol/docs/pull/181/commits/0036481e0266fdbdc98c3afb20b807a2f8bbb2e5)